doc/api/http.md

```diff
@@ -3148,13 +3148,13 @@ try {
 }
 ```
 
-[`--insecure-http-parser`]: cli.md#cli_insecure_http_parser
-[`--max-http-header-size`]: cli.md#cli_max_http_header_size_size
 [`'checkContinue'`]: #http_event_checkcontinue
 [`'finish'`]: #http_event_finish
 [`'request'`]: #http_event_request
 [`'response'`]: #http_event_response
 [`'upgrade'`]: #http_event_upgrade
+[`--insecure-http-parser`]: cli.md#cli_insecure_http_parser
```

Comment:
The `#` is not at the start of the url. Is this needed then?

Reply:
Another way of looking at it is I moved all the other references, but git prefers to show that only this line moved. Anyway, we want all references to be in some order, starting with a `#` or not ;)